kayenta-signalfx/src/integration-test/resources/config/kayenta.yml

```diff
@@ -19,6 +19,7 @@ kayenta:
     - name: in-memory-store
       supportedTypes:
       - OBJECT_STORE
+      - CONFIGURATION_STORE
```

Comment:
this and the other .yml don't seem to be part of docs

Reply:
Those where edits I made for [docs/configuring-kayenta.md](https://github.com/Nike-Inc/kayenta/blob/docs/kayenta_standalone/docs/configuring-kayenta.md) which is part of the PR.

In that doc I say that at least one configuration store is required but then link to a yml that didn't have one configured.

It's technically only required if you hit the `/config` endpoints which the integration test yml that I linked to does not.

The other yaml aka the big kayenta.yaml, I added a config option that was missing.
I think that yml isn't actually used and is more or less documentation itself.